backend/src/db/courses.js

```diff
@@ -90,6 +90,8 @@ const getCourseById = (id) => {
 
 const reduceCoursesByDate = (courses) => {
     return courses
+        .filter((course) => course.location[0])
```

Comment:
What is the purpose of these checks? What is the `course.location[0]`? Some kind of comment would be nice or then more explicit check like `course.location[0] !== null`.